md/database-configuration.md

```diff
@@ -214,9 +214,9 @@ Here is the list of steps to perform (as an example, the database name BONITA is
 12. Select and right-click on _**Local DTC**_ and then _**Properties**_.
 13. Click on _**Security**_ tab. Ensure that the checkbox for **Enable XA Transactions** is checked.
 14. Click _**Apply**_, then click _**OK**_
-15. Then stop and restart SQLServer.
-16. Create the BONITA database: `CREATE DATABASE BONITA GO`.
-17. Set `bonitadev` as owner of BONITA database (use, for example, 'Microsoft SQL Management Studio')
+15. Then stop and restart SQLServer to ensure it syncs up with the MS DTC changes.
+16. Create the BONITA\_BPM database: `CREATE DATABASE BONITA_BPM GO`.
```

Comment:
:warning: Why this change? Line 189, there is still a reference to database named `BONITA` !

Reply:
And also lines 226-229 !

Reply:
will revert to `BONITA`. this is not purpose of this PR